core/templates/dev/head/css/oppia.css

```diff
@@ -4057,21 +4057,34 @@ md-card.preview-conversation-skin-supplemental-card {
 }
 
 .oppia-about-header {
-  background-blend-mode: multiply;
-  background-color: #afd2eb;
-  background-image: url("/assets/common/images/general/icons-bg.png");
-  padding: 10%;
+  position: absolute;
   text-align: center;
+  width: 100%;
+  height: 120px;
```

Comment:
Nit: alphabetize -- i.e. height, then position, then text-align, then width. This makes it easier to find specific attrs in the future. Ditto elsewhere.